modules/ROOT/pages/backup-restore/online-backup.adoc

```diff
@@ -459,3 +459,207 @@ bin/neo4j-admin database backup --to-path=azb://myStorageAccount/myContainer/myD
 ----
 ======
 =====
+
+
+[role=label--new-2025.04]
+[[diff-backup-as-parent]]
+=== Perform a differential backup using the `--prefer-diff-as-parent` option
+
+When taking a differential backup with the `--type=DIFF` option, the parent, by default, is the *most recent non-empty* backup in the directory.
+In some cases, you may prefer to use the *latest differential* backup as the parent, even if there is a more recent full backup.
+You can do this by setting the option `--prefer-diff-as-parent` to `True`.
+
+This can be used to ensure you have differential backups for all transactions, which would allow you to restore to any point in time.
```

Comment:
I would actually move this to the top of the section and start with it. Something like:

```
You can use the `--prefer-diff-as-parent` option to ensure you have differential backups for all transactions and can restore to any point in time. 
It uses the latest differential backup as the parent, even if there is a more recent full backup. 
By default, the differential backup (`--type=DIFF`) uses the most recent non-empty backup in the directory as the parent. However, the transactions between a full backup and the previous differential backup cannot be backed up as individual transactions.

To use the --prefer-diff-as-parent option, set it to `true`.
```

Reply:
The above is my suggestion for an intro.

Reply:
It's still not perfect, but I think it should be in that line of thought.

Reply:
@renetapopova, thanks for your suggestions! I'll use it but change a little bit. I still think it's better to tell about the default behaviour first and then explain the use of the new option.

Reply:
It looks great!